megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -4181,6 +4211,15 @@ else if ((te != null) && (ae.getSwarmTargetId() == te.getId())) {
                 }
             }
         }
+        if (isBearingsOnlyMissile) {
+            //this is an arbitrary number. You shouldn't ever get this message.
+            if (distance > 5000) {
```

Comment:
Let's define the magic 5000 number somewhere relevant, since it's used in a couple of places.

Reply:
Missed a spot